### PR TITLE
Fix #252 - validate incoming /updates json, using jsonschema

### LINKED ISSUE #252: 500s on invalid request body
```
curl -v http://webapp-vmaas-stable.1b13.insights.openshiftapps.com/api/v1/updates -X POST -d '{}'
```
should result in 400. Currently it causes 500.

Related to https://github.com/RedHatInsights/insights-api/issues/207

### PR DESCRIPTION
**Pull request by ggainey**
Also adjust existing ValueError to return more useful error-info

### CODE REVIEW

File: webapp/Dockerfile [hunk 1]
```diff
@@ -9,9 +9,10 @@ ENV POSTGRESQL_PASSWORD=vmaas_reader_pwd
 ENV POSTGRESQL_DATABASE=vmaas
 
 RUN yum -y update \
+    && yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
```
Comment (grinrag) [inline webapp/Dockerfile]: Just `yum -y install epel-release`

Comment (ggainey) [reply webapp/Dockerfile]: Ah! Nice, didn't know it was already available to the image

**Comment by ggainey** [comment]
This is going to conflict hard w/the async PR #253 - wait on this one, will fix conflicts and resub once 253 is accepted

**Comment by ggainey** [comment]
Closing to resubmit against newest version of app.py in master

**Pull request closed without merging**